LICENSE_HEADER.md

```diff
@@ -58,6 +58,15 @@ rem OF ANY KIND, either express or implied. See the License for the specific lan
 rem governing permissions and limitations under the License.
 ```
 
+# Magento Projects
+
+### Text
+
+```
+Copyright © Magento, Inc. All rights reserved.
```

Comment:
Should the copyright be updated to Adobe?

Reply:
I think Magento is being retained as a separate copyright.

Reply:
Yes, chatted with @dmanners about this in the opensourceoffice Slack and we decided to stay off this for now.